src/ucp/core/ucp_worker.c

```diff
@@ -33,6 +33,24 @@ static ucs_stats_class_t ucp_worker_stats_class = {
         [UCP_WORKER_STAT_TAG_RX_RNDV_UNEXP]        = "rx_rndv_rts_unexp"
     }
 };
+
+static ucs_stats_class_t ucp_worker_offload_stats_class = {
+    .name           = "tag_offload",
+    .num_counters   = UCP_WORKER_STAT_TAG_OFFLOAD_LAST,
+    .counter_names  = {
+        [UCP_WORKER_STAT_TAG_OFFLOAD_POSTED]           = "posted",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_CANCELED]         = "canceled",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_TAG_EXCEED]       = "no_tag",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_NON_CONTIG]       = "non_contig",
```

Comment:
does it count number of tags that were NOT offloaded because the data was non-contig? if so, counter name should indicate that. since currently counter name suggests the opposite meaning. for example, "block_non_contig", or create another sub-class "block" and under it put "wildcard", "noncontig", "sw_pend"